core/config.cpp

```diff
@@ -77,15 +78,27 @@ bool Config::init_with_args(int ac, char** av, const std::vector<std::string>& c
                                      "Worker information.\nFormat is '%worker_hostname:%thread_number'.\nUse "
                                      "colon ':' as separator.");
 
+    po::options_description optional_options("Optional options");
```

Comment:
For the current arguments parser, user can easily get params from configuration files and no need to put anything in customized options. In code, just use `Context::get_param(a, default_value)`. See https://github.com/husky-team/husky/pull/264.